cli/src/commands/source.rs

```diff
@@ -0,0 +1,30 @@
+use std::path::{
```

Comment:
`source.rs` isn't a command; it's some utility thing. Can we move it into a `utils.rs` or similar please?